### ISSUE TITLE
Update tutorial

### ISSUE THREAD

**Issue by reitblatt** [frenetic-lang/frenetic#234]
Very out of date. Should replace OX section w/ ocaml-openflow section.

**Comment by craig-riecke** [comment]
OK, I have made a first run-through updating the Tutorial and the VM (the link for the VM is on the Tutorial Intro page).  The build script NetKAT syntax extensions is particularly ugly, but it works.  Next steps, which I will try to get done in the next few days:

* Update solutions code, which is kept on the master branch of frenetic-tutorials
* Remove some boilerplate code in each example (I think the startup sequence for Ox apps can be reduced to one line, as we saw in Thursday).
* Get the NetKAT reference + BNF page redone and linked in
* Get the API doc pages up and running again, and the links in the tutorial redone (since the module names have changed).
* Officially release the Ox and Syntax Extension bugfixes into Frenetic Core

The prebuilt Frenetic Tutorial VM should be a stop gap.   They should be able to run it on the official Frenetic-VM - it should take a few changes to make it happen.  It might also be nice to regen a pre-built Frenetic VM every time Frenetic master has a new commit.  A fat pipe can download the 2 GB VM faster than a full Vagrant build can run (about 30 minutes on my MacBook.)
